docs/howtos/telemetry/10-opentelemetry-qs.md

```diff
@@ -120,7 +120,7 @@ helm install --wait \
 Once cert-manager is up and running, the OpenTelemetry operator Helm chart can be installed in this way:
 
 :::note
-At time of writing the latest OpenTelemetry operator chart version is `0.56.0`
+At time of writing the latest OpenTelemetry operator chart version is `0.65.0`
```

Comment:
Date inclusion for discussion. I like the hard date here. And there is a tiny typo.

```suggestion
At the time of writing (2024-11-11) the latest OpenTelemetry operator chart version is `0.65.0`
```